ld_openfeature/provider.py

```diff
@@ -29,51 +85,52 @@ def get_provider_hooks(self) -> List[Hook]:
         return []
 
     def resolve_boolean_details(
-        self,
-        flag_key: str,
-        default_value: bool,
-        evaluation_context: Optional[EvaluationContext] = None,
+            self,
```

Comment:
Auto formatting did this. I could undo them.

Reply:
If you don't mind. That feels really over-indented to me.